### PR TITLE
Update Python.gitignore

### PR DESCRIPTION
**Pull request by 5StevenWu**
#add pycharm  .idea/
#add django settings.py

**Reasons for making this change:**

_TODO_

**Links to documentation supporting these rule changes:**

_TODO_

If this is a new template:

 - **Link to application or project’s homepage**: _TODO_

### CODE REVIEW

File: Python.gitignore [hunk 1]
```diff
@@ -58,6 +58,7 @@ coverage.xml
 # Django stuff:
 *.log
 local_settings.py
+settings.py
```
Comment (kryskool) [inline Python.gitignore]: settings.py need to be keep on the repository

File: Python.gitignore [hunk 2]
```diff
@@ -127,3 +128,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+#Pycharm
+.idea/
```
Comment (kryskool) [inline Python.gitignore]: It's ok for PyCharm

**Comment by shiftkey** [comment]
@5StevenWu thanks for opening this PR

I'm going to second the feedback from @kryskool here, and without more context about the reason behind the changes I plan to close this PR in a few days.

**Comment by RNag** [comment]
I didn't actually submit the PR, but I think its a useful change 👍 

Pycharm editor for Python leaves behind an `.idea/` folder, but ideally there's no reason to commit it to Github i believe.

**Comment by 5StevenWu** [comment]
perfect

**Comment by BrianMSheldon** [comment]
See #3451 for an optional solution

**Pull request closed without merging**